lib/roo_on_rails/railties/http.rb

```diff
@@ -28,10 +28,13 @@ class HTTP < Rails::Railtie
           app.config.middleware.use ::Rack::Deflater
         end
 
-        app.config.middleware.insert_before(
-          ActionDispatch::Cookies,
-          ::Rack::SslEnforcer
-        )
+        # Don't use SslEnforcer in test environment as it breaks Capybara
+        unless Rails.env.test?
+          app.config.middleware.insert_before(
+            ActionDispatch::Cookies,
```

Comment:
We haven't secured it yet -- it's on the list. But it will use the same cookies as the main site. Hence my question really, I can't see a good way to do it without cookies.

I'll have a quick look at supporting API mode here but as this isn't introducing cookies merely fixing a bug with test mode I don't think it needs to be part of this PR.